test/arrays/userAPI/blockOps2D.comm-none.good

```diff
@@ -0,0 +1,414 @@
+Block-distributed array
+----------------
+eltType is: real(64)
+idxType is: int(64)
+rank is: 2
+
+size is: 100
+numElements is: 100
+shape is: (10, 10)
+
+X is:
+1.0 2.0 3.0 4.0 5.0 6.0 7.0 8.0 9.0 10.0
+11.0 12.0 13.0 14.0 15.0 16.0 0.0 0.0 0.0 0.0
```

Comment:
Huh... is the implication here that if a file doesn't have enough values to fill an array, we just stop reading and leave the rest uninitialized?  I'd have expected this to result in an error...  It'd also be fine with me to beef up the size of the file such that we can put true interesting values into every location (in which case I think the current cases will just stop reading early, which seems fine.